app/src/main/java/se/nullable/flickboard/model/KeyM.kt

```diff
@@ -208,6 +208,11 @@ sealed class Action {
         override fun visual(modifier: ModifierState?): ActionVisual =
             ActionVisual.Icon(R.drawable.baseline_emoji_emotions_24)
     }
+
+    data object EnableVoiceMode : Action() {
+        // TODO - Do the SVGs come from some existing set? Does it have a microphone we can use?
+        override fun visual(modifier: ModifierState?): ActionVisual = ActionVisual.None
```

Comment:
I'm not sure if the other icons in the app were hand-made or were imported from some library. I cannot draw to save my life but I'm happy to update the PR if pointed to a usable microphone SVG

Reply:
The icons are just material icons, from Android Studio's "import clip art" tool.

Reply:
Perfect, thanks! I'm out of time on this for today but will play with it as I'm able